### ISSUE TITLE
Not working for local pages (DDEV hosted, within WSL2)

### ISSUE THREAD

**Issue by thomasfrobieter** [blaise-io/live-reload#27]
I am running a page using DDEV in WSL2 i call them in Firefox (DEV) on Windows 10, like so:

http://d9-theme.ddev.site

I've set up the watcher like this:

Host-URL: http://d9-theme.ddev.site/*

Source-File-URLs:
http://d9-theme.ddev.site/themes/custom/digico/css/*.css
http://d9-theme.ddev.site/themes/custom/digico/js/*.js

I am able to open "http://d9-theme.ddev.site/themes/custom/digico/css/digico.global.min.css" so the configured paths should be correct.

I would really love to see this working.. gulp-livereload, what i've used in the past, didn't work for me withing WSL2.

**Comment by blaise-io** [comment]
Hey Thomas, please include a log. 

You can view the log by visiting about:devtools-toolbox?type=extension&id=live-reload%40blaise%2Eio in a new tab.

Using WSL2 shouldn't matter. The only requirements are that source files are accessible over http(s) and to-be-monitored source files are loaded by the host page.

**Comment by thomasfrobieter** [comment]
Hey,

it does recognize changes, but they're some errors in the log:

```
Incoming message from  moz-extension://bc9da989-df53-4373-b45c-80b8126459f2/popup.html 
Object { type: "isMonitoring?" }
background.js:133:13
Storage data 
Object { isMonitoring: true, rank0284x3: {…} }
popup.js:220:17
Rules 
Array [ {…} ]
popup.js:226:17
**Error: Please use $(ref:runtime.getURL).**
**can't access property "enabled", state is null 2 global.js:344**
5 Navigated to new URL: http://d9-theme.ddev.site/themes/custom/digico/css/digico.global.min.css?qvkbf1= background.js:122:17
5 disable tab monitoring background.js:249:13
5 http://d9-theme.ddev.site/themes/custom/digico/css/digico.global.min.css?qvkbf1= matches rule Digitalcourage DDEV background.js:184:21
5 http://d9-theme.ddev.site/themes/custom/digico/css/digico.global.min.css?qvkbf1= initialize monitoring background.js:194:21
SKIP http://d9-theme.ddev.site/themes/custom/digico/css/digico.global.min.css?qvkbf1= background.js:214:17
5 disable tab monitoring background.js:249:13
5 remove webrequests listener background.js:266:17
**can't access property "enabled", state is null**
```

Still nothing changes, no reload. I've tried to disable "Reload stylesheets without reloading the host page" but no difference.

**Comment by blaise-io** [comment]
Possibly fixed in fbfce2400be43dc82ee33fb0baf04891045341bf and Live Reload 1.8.3. Can you verify?

On my machine `Error: Please use $(ref:runtime.getURL)` seems to be non-fatal, so I'm not 100% sure this fixes your issue.

**Comment by blaise-io** [comment]
I'll consider this fixed.
